docs/contributor/data.md

```
@@ -652,20 +702,33 @@ Result:          ❌ NO MATCH - Recipe requires gb200, query has "any" (wildcard
 
 This asymmetric behavior ensures that a generic query like `--service eks --intent training` only matches generic recipes, not hardware-specific ones like `gb200-eks-training.yaml`.
 
-**Example 4: Multiple Matches (Fully Specific Query)**
+**Example 4: Multiple Maximal Matches (Fully Specific Query)**
 ```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)